### PR TITLE
VACMS-3467: CLP Video Panel

### PR DESCRIPTION
**Pull request by kaise-lafrai**
## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3467

## QA steps

As user _uid_ with _user_role_

- Login to PR environment
- Navigate here:` /media/add/video`
- Confirm the the following:
  - [x] Video URL *
  - [x] Name *
  - [x] Duration `h:mm:ss`
  - [x] Description `5000 characters limit`
- Complete **Add Video** form and save
  - [x] Save successful
proof: 
image

- Navigate here: /node/add/campaign_landing_page
- Select the **Video** panel tab
- Confirm the the following:
  - [x] Add a video panel `boolean`
  - [x] Panel Header `70 character limit`
  - [x] Video Entity Reference `Add media`
  - [x] Add a link to more videos `Button` with Button Label and Button Link
- Select Add media button
  - [x] Confirm the new video added from steps above appears and Duration and Description are visible.
- Complete all required fields on other tabs
  - [x] Save successful

- [x] confirm panel toggle follows pattern from stories.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by kaise-lafrai** [comment]
@kevwalsh I'm still writing up the QA steps. Please feel free to check the work out.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
A couple possible issues.

File: config/sync/core.entity_form_display.media.video.media_library.yml [hunk 1]
```diff
@@ -71,9 +73,10 @@ content:
     region: content
 hidden:
   created: true
+  field_description: true
+  field_duration: true
   moderation_state: true
   path: true
   preview: true
-  revision_log_message: true
```
[the remark this answers is not in the document]
Comment (acrollet) [reply config/sync/core.entity_form_display.media.video.media_library.yml:77]: hmm, I guess our revision log test doesn't catch these cases?

**Review by swirtSJW — COMMENTED** [review]
Code looks good.   Testing locally now.

**Comment by swirtSJW** [comment]
Needs to be connected to Ethan's hidey logic but can be in followup ticket 
image

**Review by kevwalsh — APPROVED** [review]
Screenshot of the video panel form looks good to me! 

@swirtSJW i tested the video bundle form previously, so assuming it hasn't changed this is good to go. i can't test locally right now. 

Boolean form hiding logic will take place in #3575, next sprint

**Comment by swirtSJW** [comment]
Proof of working 
image


I completed #3575  too.

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
Devshop completely fell down on building this 
image

Merging based on tests passing on sandbox.

Nice work @kaise-lafrai

**Pull request merged**